buildenv/jenkins/openjdk_tests

```diff
@@ -223,51 +233,81 @@ if (JDK_VERSIONS.size() > 1 || JDK_IMPLS.size() > 1 || PLATFORMS.size() >1 || PL
         println "LABEL: ${LABEL}"
 
         stage('Queue') {
-            if (nodesByLabel(LABEL).size() < 1) {
+            String[] onlineNodes = nodesByLabel(LABEL)
+            if (onlineNodes.size() < 1) {
                 int ACTIVE_NODE_TIMEOUT = params.ACTIVE_NODE_TIMEOUT ? params.ACTIVE_NODE_TIMEOUT : 0
-
                 timeout(ACTIVE_NODE_TIMEOUT) {
-                    // node(LABEL) has to be used for spinning up a dynamic vm agent
+                    // If there is available node before timeout
                     node(LABEL) {
-                        echo "find the node ${env.NODE_NAME}"
+                        echo "find the node with label as ${env.NODE_NAME}"
+                    }
+                }
+            } else {
+                boolean isNodeIdle = false
+                node('master') {
```

Comment:
Eclipse OpenJ9 Jenkins does not have any executors on master so this won't work there.